test/config-luatest/CMakeLists.txt

```diff
@@ -23,21 +17,10 @@ foreach(test_path ${tests})
   if(${test_name} STREQUAL ${TEST_SUITE_NAME})
     continue()
   endif()
-  set(test_title "test/${TEST_SUITE_NAME}/${test_name}")
-  add_test(NAME ${test_title}
-           COMMAND ${TARANTOOL_BIN}
-                   # Disable stdout buffering,
-                   # see https://github.com/tarantool/test-run/issues/119.
-                   -e "io.stdout:setvbuf('no')"
-                   ${LUATEST_COMMAND_WITH_FLAGS}
-                   --shuffle all:${RANDOM_SEED}
-                   ${test_title}
-           WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}
-  )
-  set_tests_properties(${test_title} PROPERTIES
-    ENVIRONMENT "${TEST_ENV}"
-    LABELS "${TEST_SUITE_NAME};regression;"
-    DEPENDS ${TEST_SUITE_NAME}-deps
-    FAIL_REGULAR_EXPRESSION "${CTEST_FAIL_REGEXP}"
-  )
+  add_luatest_test(${TEST_SUITE_NAME} ${test_name} TRUE)
 endforeach()
+
+set_tests_properties(
+    test/config-luatest/jsonschema_config_test.lua
+    PROPERTIES DISABLED TRUE
```

Comment:
Please add a comment with reason and followup issue.

Reply:
Of course, I'll do this. Note that this PR is a draft. It is not ready for review yet.

Reply:
Filed https://github.com/tarantool/tarantool/issues/11728. Will add a reference to this issue in the code before submitting this PR for review.